brain-marks/Categories/Views/CategoryRow.swift

```diff
@@ -8,15 +8,16 @@
 import SwiftUI
 
 struct CategoryRow: View {
-    let category: AWSCategory
+    let categoryName: String
```

Comment:
I'd rather keep this as passing a single category object, rather than 2 properties, because it's possible for this to grow

Reply:
Done ✅